libs/components/select-field/.eslintrc.json

```diff
@@ -36,7 +36,9 @@
         "../../../.eslintrc-overrides.json",
         "prettier"
       ],
-      "rules": {}
+      "rules": {
+        "@angular-eslint/template/prefer-control-flow": ["warn"]
```

Comment:
do we need this here?

Reply:
Yes. Select field is deprecated and we aren't going to update it. This will allow that library to continue passing CI once we turn the ESLint rule to `error` after everything is converted.